index.php

```diff
@@ -1,22 +0,0 @@
-<html>
-  <head>
-    <meta charset="utf-8">
-    <title>Sign Up For Beta Form</title>
-    <link rel="stylesheet" href="css/style.css">
-    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
-  </head>
-  <body>
-    <form action="thankyou.php" method="post" name="sign up for beta form">
-      <div class="header">
-         <p>Sign Up For XKCD Comic updates</p>
-      </div>
-      <div class="description">
-        <p>Enter your email address to get a verification mail and start recieving your Comic Updates</p>
-      </div>
-      <div class="input">
-        <input type="email" class="button" id="email" name="email" placeholder="NAME@EXAMPLE.COM">
```

Comment:
Why does a text input have class `button`?

Reply:
i tried sending html mail with a button to verify the user i thought it would look good